enterprise/ha/src/main/java/org/neo4j/kernel/impl/ha/ClusterManager.java

```diff
@@ -123,8 +124,20 @@
         HighlyAvailableGraphDatabase repair() throws Throwable;
     }
 
+    public static IntFunction<String> constant( final String value )
+    {
+        return new IntFunction<String>()
+        {
+            @Override
+            public String apply( int ignored )
+            {
+                return value;
+            }
+        };
+    }
+
     private final File root;
-    private final Map<String,String> commonConfig;
+    private final Map<String,IntFunction<String>> commonConfig;
```

Comment:
I wanted to unify common config and per-instance config... less code ftw. But I could split them apart if that makes more sense.